kube-deploy/group_vars/all.yml

```diff
@@ -18,6 +18,8 @@ bootstrap_os: ubuntu
 public_iface: enp0s3
 nat_iface: enp0s3
 proxy_enable: false
+#Only enable this if you are deploying on ARM or ARM64 with Fannel SDN provider
+arm_enabled: false
```

Comment:
i would strongly suggest using `facts` based on the CPU for `arm64`. we can always  take a stab at it, and let @ravishivt try it. we can also see if @luxas would be willing to try this. @aric49 if all else fails...I will bring the _infamous_ Pi Cluster to the office,  and you can use that to test with. :)